### PR TITLE
#392 Sauter's formulae for plasma current and geometric parameters.

### PR DESCRIPTION
**Pull request by mkovari**
Minor refactor of `culcur` in `physics.py`, as described in #392:
Update docstring.
Rename qpsi in culcur as q95.
Eliminate curhat since it's not necessary.
Test for a range of values of triangularity.

### CODE REVIEW

**Comment by mkovari** [comment]
`E       TypeError: culcur() got an unexpected keyword argument 'qpsi'`
I have renamed `qpsi` in culcur as `q95`.
@timothy-nunn

**Comment by mkovari** [comment]
See detailed comments in the issue #392.

File: process/plasma_geometry.py [hunk 1]
```diff
@@ -534,3 +553,40 @@ def xsect0(self, a, kap, tri):
         xsect0 = xlo**2 * (thetao - cto * sto) + xli**2 * (thetai - cti * sti)
 
         return xsect0
+
+    def Sauter_geometry(self, a, r0, kap, tri):
```
Comment (timothy-nunn) [inline process/plasma_geometry.py:557]: Even though its a persons name, I think `Sauter` should be lower case

File: process/plasma_geometry.py [hunk 2]
```diff
@@ -249,10 +233,45 @@ def geomty(self):
             thetao,
         )
         physics_variables.sareao = xso
-        physics_variables.sarea = xsi + xso
 
-        #  Cross-sectional area
-        physics_variables.xarea = self.xsecta(xi, thetai, xo, thetao)
+        # icurr = 8 specifies use of the Sauter geometry as well as plasma current.
```
Comment (timothy-nunn) [inline process/plasma_geometry.py:237]: Will it ever make sense to use Sauter geometry without using the Sauter plasma current?

**Review by timothy-nunn — CHANGES_REQUESTED** [review]
One minor change for code quality, one question to ensure this change is utilised to its fullest potential

File: process/plasma_geometry.py (hunk 2 above)
Comment (timothy-nunn) [reply process/plasma_geometry.py:237]: Ok. If you make this into a new issue we can discuss it there

**Pull request merged**